content/post/2023/captioning-tools-extravaganza.md

```diff
@@ -0,0 +1,127 @@
+
+### As a Presenter, I want captioning, so that my talk is accessible
+
+Generally, the usual advice for speaking applies. Speak as clearly as you can, avoid cross-talk if you're co-presenting with someone. If you're doing a Q&A session, it's helpful to repeat the question back to make sure it shows up in the captioning feed.
+
+If you're able, turn on captions while you're doing practice runs for the talk. This should give you a sense of how accurate these tools are and how they sometimes misunderstand what you say. If your talk uses uncommon tech jargon, it's helpful to know how it may come up on screen.
+
+#### Windows Live Captions
+- **Free, requires Windows 11 22H2 or newer, works offline**
+
+_My thoughts_: Quick and pretty accurate, deals well with accents. Can be a little glitchy with repeating fragments of a sentence that's not finalized. Includes a profanity filter. I use this heavily every single day on both my work and home machines, and it's tremendously useful. This requires downloading several hundred megabytes of supporting files, so be sure to set it up ahead of time.
+
+#### PowerPoint Live Captions
+- **Free, included in PowerPoint, cloud-based**
+
+_My thoughts_: I've seen one or two presentations that used this, and it seems to work well. It uses a cloud-based service, so you will need a stable internet connection for it to work. Also, the captions may stop, or at least won't be visible, if you switch out of PowerPoint to do a live demo of something.
+
+#### Google Slides Live Captions
+- **Free, included in Google Slides**
+
+_My thoughts_: I haven't used this one personally; one of the draft reviewers mentioned it. Google Meet's built-in captions are high quality in my experience, so I would expect these to be on a similar level. Experiment first.
+
+#### Apple Live Captions
+- **Free, requires macOS 13 Ventura or newer, requires Apple Silicon, works offline**
+- **Also available on iOS 16 or newer, requires iPhone 11 or newer**
+- **Also available on iPadOS 16 or newer, [see instructions for supported devices](https://support.apple.com/guide/ipad/get-live-captions-beta-ipad0bbca12e/ipados)**
+- **Does not work if screen recording (including DisplayLink) is active**
+
+_My thoughts_: I appreciate that Apple added this. Unfortunately, I think it is the weakest of the available options. In my experience, it struggles more with accuracy and with technical vocabulary than other options do, but it's absolutely work trying it out for yourself. Also, note that depending on how your machine is set up, it may not function - Apple disables it by rendering a gray box over the captions window if any sort of screen recording software is active. This, unfortunately, includes when you are using DisplayLink adapters to get multiple external displays on M1/M2 Macs that wouldn't otherwise support it; DisplayLink on the Mac is implemented using screen recording.
```

Comment:
```suggestion
_My thoughts_: I appreciate that Apple added this. Unfortunately, I think it is the weakest of the available options. In my experience, it struggles more with accuracy and with technical vocabulary than other options do, but it's absolutely worth trying it out for yourself. Also, note that depending on how your machine is set up, it may not function - Apple disables it by rendering a gray box over the captions window if any sort of screen recording software is active. This, unfortunately, includes when you are using DisplayLink adapters to get multiple external displays on M1/M2 Macs that wouldn't otherwise support it; DisplayLink on the Mac is implemented using screen recording.
```